src/main/java/it/pagopa/pn/deliverypush/action/utils/AarUtils.java

```diff
@@ -34,6 +37,12 @@ public AarUtils(TimelineService timelineService, TimelineUtils timelineUtils, Sa
     }
 
     public void generateAARAndSaveInSafeStorageAndAddTimelineevent(NotificationInt notification, Integer recIndex, String quickAccessToken) {
+        PnAuditLogBuilder auditLogBuilder = new PnAuditLogBuilder();
```

Comment:
la creazione dell'audit log va dentro l'if (timeline.isEmpty)

Reply:
Inoltre, mi son dimenticato, ho creato il service AuditLogService, che si occupa di creare il logEvent e loggarlo.
Utile perchè in fase di test, si può mockare e controllare che il codice passi per generatesuccess/failure . Sarebbe da usare